cpp/oneapi/dal/algo/shortest_paths/test/shortest_paths_test.cpp

```diff
@@ -17,9 +17,937 @@
+    std::vector<double> edge_weights = { 1, 1, 1, 1 };
+    std::vector<float> distances = { 0,
+                                     std::numeric_limits<float>::infinity(),
+                                     std::numeric_limits<float>::infinity(),
+                                     std::numeric_limits<float>::infinity(),
+                                     std::numeric_limits<float>::infinity() };
+};
+
+class d_k_15_double_edges_source_5_type : public graph_base_data {
+public:
+    d_k_15_double_edges_source_5_type() {
+        vertex_count = 15;
+        edge_count = 210;
+        cols_count = 210;
+        rows_count = 16;
+        source = 5;
+    }
+    std::vector<std::int64_t> rows = { 0,   14,  28,  42,  56,  70,  84,  98,
+                                       112, 126, 140, 154, 168, 182, 196, 210 };
+    std::vector<std::int32_t> cols = {
+        1, 2, 3,  4,  5,  6,  7,  8, 9, 10, 11, 12, 13, 14, 0, 2, 3,  4,  5,  6,  7,
+        8, 9, 10, 11, 12, 13, 14, 0, 1, 3,  4,  5,  6,  7,  8, 9, 10, 11, 12, 13, 14,
+        0, 1, 2,  4,  5,  6,  7,  8, 9, 10, 11, 12, 13, 14, 0, 1, 2,  3,  5,  6,  7,
+        8, 9, 10, 11, 12, 13, 14, 0, 1, 2,  3,  4,  6,  7,  8, 9, 10, 11, 12, 13, 14,
+        0, 1, 2,  3,  4,  5,  7,  8, 9, 10, 11, 12, 13, 14, 0, 1, 2,  3,  4,  5,  6,
+        8, 9, 10, 11, 12, 13, 14, 0, 1, 2,  3,  4,  5,  6,  7, 9, 10, 11, 12, 13, 14,
+        0, 1, 2,  3,  4,  5,  6,  7, 8, 10, 11, 12, 13, 14, 0, 1, 2,  3,  4,  5,  6,
+        7, 8, 9,  11, 12, 13, 14, 0, 1, 2,  3,  4,  5,  6,  7, 8, 9,  10, 12, 13, 14,
+        0, 1, 2,  3,  4,  5,  6,  7, 8, 9,  10, 11, 13, 14, 0, 1, 2,  3,  4,  5,  6,
+        7, 8, 9,  10, 11, 12, 14, 0, 1, 2,  3,  4,  5,  6,  7, 8, 9,  10, 11, 12, 13
+    };
+    std::vector<double> edge_weights = {
+        69, 90, 91,  83, 35, 71, 67, 25, 78, 83, 90, 26,  47, 23, 92,  61, 33, 95,  44, 63, 22,
+        51, 11, 41,  33, 25, 56, 83, 39, 26, 58, 63, 100, 67, 47, 45,  27, 19, 55,  35, 51, 10,
+        66, 96, 95,  36, 21, 48, 32, 60, 51, 33, 22, 84,  34, 13, 94,  50, 32, 38,  75, 64, 98,
+        56, 93, 95,  54, 24, 45, 28, 60, 60, 85, 27, 99,  30, 65, 24,  69, 70, 43,  46, 66, 52,
+        16, 38, 50,  94, 64, 85, 25, 53, 79, 87, 90, 34,  45, 96, 53,  54, 34, 73,  32, 47, 16,
+        50, 37, 69,  81, 49, 53, 68, 61, 23, 64, 88, 35,  62, 24, 81,  41, 59, 100, 83, 26, 100,
+        94, 23, 100, 62, 54, 24, 34, 92, 76, 24, 66, 24,  73, 89, 17,  45, 72, 22,  71, 19, 55,
+        72, 59, 87,  63, 16, 98, 35, 53, 93, 54, 53, 10,  73, 76, 100, 12, 63, 94,  11, 47, 34,
+        35, 73, 94,  99, 34, 39, 11, 39, 47, 73, 54, 93,  54, 25, 75,  95, 51, 83,  76, 18, 79,
+        24, 89, 76,  24, 73, 93, 13, 11, 51, 54, 49, 23,  28, 21, 16,  98, 40, 13,  26, 11, 23
+    };
+    std::vector<float> distances = { 46, 47, 80, 27, 53, 0, 30, 55, 24, 58, 53, 43, 46, 50, 40 };
+};
+
+template <class T>
+struct LimitedAllocator {
+    typedef T value_type;
+    typedef T* pointer;
```

Comment:
unused?

Reply:
Check std allocator traits requirements for the allocator and cover the only required things